flutter_app/lib/drone_information.dart

```diff
@@ -0,0 +1,109 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:imacs/mavlink_communication.dart';
+
+// DroneInformation is a widget that displays all the data fetched from
+// Mission Planner MAVLink. It automatically sets the data in a two
+// column layout.
+class DroneInformation extends StatelessWidget {
+  const DroneInformation({
```

Comment:
Add documentation for the constructor, look at `data_field_widget.dart` for an example